src/main/java/org/usfirst/frc4904/robot/commands/indexer/IndexerOff.java

```diff
@@ -0,0 +1,20 @@
+package org.usfirst.frc4904.robot.commands.indexer;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+
+public class IndexerOff extends ParallelCommandGroup {
+
+  /**
+   * Set indexer motor speed to zero
+   */
+  public IndexerOff() {
+    this.addCommands(
+      new MotorIdle(RobotMap.Component.indexer.indexerMotor1),
```

Comment:
inconsistent indentation amount—4 spaces per tab is probably better here than two spaces per tab